src/styles/styles.css

```diff
@@ -0,0 +1,135 @@
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 200px;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+  padding: 16px;
+  font-family: Roboto, sans-serif;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  margin-top: 16px;
+}
+
+.card__item-model {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: rgb(6, 11, 53);
+  box-sizing: border-box;
+  width: 100%;
+  margin-top: 40px;
+}
+
+.card__item-code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: rgb(97, 96, 112);
+  box-sizing: border-box;
+  width: 100%;
+  text-align: left;
+  margin-top: 4px;
+}
+
+.card__item-review {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  box-sizing: border-box;
+  width: 100%;
+  margin-top: 16px;
+}
+
+.card__review {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: rgb(6, 11, 53);
+  clear: both;
+  display: inline-block;
+  white-space: nowrap;
+}
+
+.card__item-price {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+  box-sizing: border-box;
+  width: 100%;
+  margin-top: 24px;
+}
+
+.card__item-price p:first-child {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: rgb(97, 96, 112);
+}
+
+.card__item-price p:last-child {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 16px;
+  color: rgb(6, 11, 53);
+}
+
+.card__button {
+  margin-top: 16px;
+  height: 40px;
+  width: 100%;
+  border-radius: 5px;
+  background-color: rgb(0, 172, 220);
+  border: 1px solid rgb(0, 172, 220);
+  box-sizing: border-box;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  color: white;
+}
+
+.card__button:hover {
+  background-color: rgb(243, 243, 243);
+  color: rgb(0, 172, 220);
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  width: 100%;
+}
+
+.stars__star {
+  background-image: url('../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  padding: 0;
+}
+
+.stars--1 .stars__star:first-child,
+.stars--2 .stars__star:nth-last-child(n + 4),
+.stars--3 .stars__star:nth-last-child(n + 3),
+.stars--4 .stars__star:nth-last-child(n + 2),
+.stars--5 .stars__star:nth-last-child(n + 1) {
+  background-image: url('../images/star-active.svg');
```

Comment:
Similarly, verify that the path to the image '../images/star-active.svg' is correct relative to the location of this CSS file.